src/ORM/Table.php

```diff
@@ -1512,6 +1513,24 @@ public function save(EntityInterface $entity, $options = [])
     }
 
     /**
+     * Try to save an entity and fail if save was not successful
```

Comment:
This does not explain what `fail` is. Better say that it will throw a `PersistenceFailedException` and in what cases (entity with errors, failed application rules checking, saved aborted by callbacks)